Assets/Scripts/Enemies.cs

```diff
@@ -37,4 +37,14 @@ virtual protected void Die()
         Destroy(gameObject);
     }
 
+    virtual public void IsBeingGrabbed()
```

Comment:
Add a bool parameter and set it to isStopped

Reply:
Done